geomesa-tools/README.md

```diff
@@ -358,11 +358,13 @@ The file type is inferred from the extension of the file, so ensure that the for
         -i, --instance
            Accumulo instance name
         -lat, --lat-attribute
-           name of the latitude field in the SimpleFeatureType if ingesting point
-           data
+           name of the latitude field in the SimpleFeature if longitude is 
```

Comment:
Change line to be 
"name of the latitude field in the SimpleFeature if latitude is"